### PR TITLE
PR: bridge communication between UbiquityOS & PersonalAgent Plugin

### PR DESCRIPTION
**Pull request by EresDev**
PR closed in favor of https://github.com/EresDevOrg/personal-agent-bridge/pull/3

### CODE REVIEW

File: .github/workflows/compute.yml [hunk 1]
```diff
@@ -20,12 +20,12 @@ on:
...
   compute:
-    name: "plugin name"
+    name: "Personal Agent Bridge"
     runs-on: ubuntu-latest
     permissions: write-all
     env:
-      SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
-      SUPABASE_KEY: ${{ secrets.SUPABASE_KEY }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      PA_BRIDGE_X25519_PRIVATE_KEY: ${{ secrets.PA_BRIDGE_X25519_PRIVATE_KEY }}
```
Comment (0x4007) [inline .github/workflows/compute.yml]: Why not just use the same key we already have available everywhere? Why are you requiring a new key for this?

I know it's generally considered best practice to make dedicated keys for each application but both would be stored in the same way essentially in cloudflare secrets or GitHub secrets.

**Comment by 0x4007** [comment]
Perhaps we should stick with normal tag syntax instead of prefixing with a slash as it seems less natural to invoke. 

---

Also I think you should host your personal agent and we can make a new task to turn it into a neat template. But for now to QA this plugin/bridge it is necessary to show both pieces.

File: .github/workflows/compute.yml (hunk 1 above)
Comment (EresDev) [reply .github/workflows/compute.yml]: > Why not just use the same key we already have available everywhere? Why are you requiring a new key for this?

You are right. I have mistaken that `X25519_PRIVATE_KEY` enforces usage in the same organization. But I checked and saw that it was not `X25519_PRIVATE_KEY` but ethreum encrypted private key that enforces usage in the same organization. So, I will use the same `X25519_PRIVATE_KEY`.

**Comment by EresDev** [comment]
> Perhaps we should stick with normal tag syntax instead of prefixing with a slash as it seems less natural to invoke.

I couldn't find such a function in my PR. Maybe you are referringg to variables in `src/helpers/keys.ts`. I took that function from permit-generation plugin as variable names are same as there. But I understand, and I will change it. 

> Also I think you should host your personal agent and we can make a new task to turn it into a neat template. But for now to QA this plugin/bridge it is necessary to show both pieces.

Yes, it is also present. 
Same user: https://github.com/EresDevOrg/personal-agent
Different user: https://github.com/web4er/personal-agent

You can also see there personal-agent workflows being called by the bridge. For now, it is simple hello-world. 
I am also adding or removing these forks for testing because I need to cover the presence and absence of these forks. So, maybe the personal-agent fork is no more by the time you see. 

I also plan to open a new PR instead of this because I want to squash/rebase some commits which helped testing by hardcoding some stuff. This PR will stay too. But a new PR with cleaner commit history will be main one.

**Comment by 0x4007** [comment]
Cool. Looking forward one day to write some sequence of characters and have an LLM rewrite my comment to be more clear. Very useful from mobile!

**Pull request closed without merging**